pkg/virt-controller/watch/vm.go

```diff
@@ -2597,7 +2619,7 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 				return vm, nil, err
 			}
 		} else {
-			err = c.stopVMI(vm, vmi)
+			err = c.stopVMI(vm, vmi, nil)
```

Comment:
I would have expected here that the value from `vmi.spec.terminationGracePeriodSeconds` would be taken.
Can you explain why `nil` is used here? Thanks

Reply:
> This is because the default behavior of Delete, when `GracePeriodSeconds` is nil, will use the object's default grace-period, in this case the grace-period stored in `vmi.spec.TerminationGracePeriodSeconds`

Sorry for being pedantic, but I am not able find this behavior anywhere. I mean, I think that if not specified, the default grace period used will be the one of the pod object ([30 sec](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L3354-L3363)) as descrihbed here: https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L497-L502

Reply:
Ah I see what you mean. I will go ahead and update it then to use `vmi.spec.TerminationGracePeriodSeconds` then.